idea_town/frontend/static-src/app/collections/experiments.js

```diff
@@ -13,6 +15,19 @@ export default Collection.extend({
   // updated yet? TODO: investigate
   ajaxConfig: { headers: { 'Accept': 'application/json' }},
 
+  initialize() {
+    // TODO:(DJ) This is not being run on the experiment-page.
```

Comment:
Ah, okay, I think this might be a TODO that corresponds to what I'm seeing... if I load the home page, then click through to an installed experiment, the button says "disable". But, if I reload the experiment page, the button flips back to "enable" even though the experiment is already installed.

Probably want to file an issue for this TODO, but can probably just merge this and move forward